modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js

```diff
@@ -572,11 +572,7 @@ const PriceRows = ({price}) => {
 						priceName={Liferay.Language.get('price-as-configured')}
 					>
 						<span className="text-7">
-							{hasDiscountPercentage
-								? price.finalPriceFormatted
-								: hasPromoPrice
-								? price.promoPriceFormatted
-								: price.priceFormatted}
+							{price.finalPriceFormatted}
```

Comment:
I don't get why you need to remove this. How can you show the promoprice? and what was the priceFormatted , why that is removed as well?

Reply:
It should be fine now because the final price is always filled in the APIs and primo price has its own row